pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -401,10 +401,12 @@ impl CustomScan for PdbScan {
                 0
             };
 
+            // TODO: Re-examine this `is_string_fast_field_capable` check after #2612 has landed,
+            // as it should likely also be checking for `is_mixed_fast_field_capable` as well, and
+            // should likely have different thresholds.
```

Comment:
IMHO, it's better to rebase this branch on top of #2612, test this and remove this comment?